Chethz/Pacman/PacmanTests/PacmanTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿using NUnit.Framework;
+using PacmanLibrary;
+
+namespace PacmanTests
+{
+    public class PacmanTests
+    {
+
+        private Pacman _pacman;
+        private Grid _grid;
+
+        [SetUp]
+        public void Setup()
+        {
+            _pacman = new Pacman();
+            _grid = new Grid(5, 5);
+            _pacman.Place(_grid, 1, 1, Directions.Face.NORTH);
+        }
+
+        [Test]
+        public void Place_SetGrid_ShouldReturnNotNull()
+        {
+            Assert.IsNotNull(_pacman.Grid);
+        }
+
+        [Test]
+        public void Place_WhenPlaceOutsideGrid_ShouldIgnoreCommand()
+        {
+            var pacman = new Pacman();
+            pacman.Place(6,6, Directions.Face.NORTH);
+            Assert.IsNull(pacman.Grid);
+        }
+
+        [Test]
+        public void Place_WhenPlaceOutsideGrid_ShouldIgnoreCommandStayInTheSameCoordinates()
+        {
+            _pacman.Place(6, 6, Directions.Face.NORTH);
+            Assert.That(_pacman.X, Is.EqualTo(1));
+            Assert.That(_pacman.Y, Is.EqualTo(1));
+            Assert.That(_pacman.Face, Is.EqualTo(Directions.Face.NORTH));
+        }
```

Comment:
Should really be two tests - one for testing when X is out of bounds, and one for when Y is out of bounds